Airports/EGGW/SMR/Labels.txt

```diff
@@ -1,109 +1,111 @@
-"R" N051.52.51.030 W000.22.47.180 standHold ;stand 12R
-"12" N051.52.50.860 W000.22.47.590 standHold
-"R" N051.52.49.930 W000.22.48.580 standHold ;stand 13R
-"13" N051.52.49.520 W000.22.49.270 standHold
-"L" N051.52.48.830 W000.22.49.980 standHold ;stand 13L
-"14" N051.52.48.190 W000.22.50.960 standHold
-"L" N051.52.47.720 W000.22.51.370 standHold ;stand 14L
-"15" N051.52.46.780 W000.22.52.490 standHold
-"L" N051.52.46.620 W000.22.52.770 standHold ;stand 15L
-"16" N051.52.38.950 W000.22.47.430 standHold
-"17" N051.52.37.710 W000.22.46.190 standHold
-"18" N051.52.37.540 W000.22.43.080 standHold
-"19" N051.52.37.370 W000.22.41.310 standHold
-"20" N051.52.35.720 W000.22.24.510 standHold
-"21" N051.52.36.250 W000.22.22.150 standHold
-"L" N051.52.35.660 W000.22.22.670 standHold ;Stand 21L
-"R" N051.52.36.340 W000.22.20.670 standHold ;Stand 21R
-"22" N051.52.36.860 W000.22.18.600 standHold
-"23" N051.52.37.460 W000.22.15.110 standHold
-"L" N051.52.37.080 W000.22.16.390 standHold ;Stand 23L
-"R" N051.52.37.560 W000.22.14.300 standHold ;Stand 23R
-"30" N051.53.03.470 W000.22.26.310 standHold
-"31" N051.53.03.850 W000.22.21.680 standHold
-"R" N051.52.53.026 W000.22.29.354 standHold ;Stand 41R
-"41" N051.52.53.020 W000.22.28.080 standHold
-"L" N051.52.53.610 W000.22.27.220 standHold ;Stand 41L
-"R" N051.52.52.810 W000.22.29.140 standHold ;Stand 42R
-"42" N051.52.53.860 W000.22.23.990 standHold
-"L" N051.52.54.470 W000.22.23.000 standHold ;Stand 42L
-"R" N051.52.53.670 W000.22.24.920 standHold ;Stand 42R
-"43" N051.52.50.680 W000.22.27.750 standHold
-"L" N051.52.50.610 W000.22.28.090 standHold ;stand 43L
-"44" N051.52.51.260 W000.22.24.920 standHold
-"L" N051.52.50.700 W000.22.25.800 standHold ;stand 44L
-"R" N051.52.51.470 W00.22.23.870 standHold ;stand 44R
-"45" N051.52.51.830 W000.22.22.090 standHold
-"R" N051.52.51.560 W000.22.21.580 standHold ;stand 45R
-"47" N051.52.46.740 W000.22.23.640 standHold
-"48" N051.52.47.170 W000.22.21.570 standHold
-"46" N051.52.46.160 W000.22.26.160 standHold
-"49" N051.52.47.530 W000.22.19.380 standHold
-"54" N051.52.41.856 W000.22.51.248 standHold
-"56" N051.52.43.180 W000.22.56.080 standHold
-"58" N051.52.47.210 W000.22.58.820 standHold
-"60" N051.52.52.750 W000.22.33.690 standHold
-"61" N051.52.53.290 W000.22.32.340 standHold
-"80" N051.52.57.100 W000.22.33.000 standHold
-"81" N051.52.55.440 W000.22.40.210 standHold
-"90" N051.52.38.716 W000.22.15.713 smrRed
-"91" N051.52.39.668 W000.22.16.076 smrRed
-"92" N051.52.41.341 W000.22.16.948 smrRed
-"93" N051.52.42.428 W000.22.17.654 smrRed
-;Tug Release Points
-"TP-V" N051.52.49.990 W000.22.21.330 smrTugPoint
-"TP-X" N051.52.49.000 W000.22.25.820 smrTugPoint
-"TP-Y" N051.52.51.760 W000.22.16.600 smrTugPoint
-"TP-Z" N051.52.46.870 W000.22.17.410 smrTugPoint
+;Tug   Release points
+
+"TP-X" N051.52.48.775 W000.22.26.324 standHold
+"TP-V" N051.52.49.732 W000.22.21.866 standHold
+"TP-Y" N051.52.51.911 W000.22.17.476 standHold
+"TP-Z" N051.52.46.954 W000.22.17.653 standHold
```

Comment:
```suggestion
;Tug Release Points
"TP-V" N051.52.49.732 W000.22.21.866 smrTugPoint
"TP-X" N051.52.48.775 W000.22.26.324 smrTugPoint
"TP-Y" N051.52.51.911 W000.22.17.476 smrTugPoint
"TP-Z" N051.52.46.954 W000.22.17.653 smrTugPoint
```
Case, Alphabteical, Whitespace
And colour for these labels should match the lines I think, as it did before, just makes them a bit clearer.